contracts/ClearingHouse.sol

```diff
@@ -850,28 +850,20 @@ contract ClearingHouse is
 
         if (response.pnlToBeRealized != 0) {
```

Comment:
We should make sure the user cannot reduce with a position size that exceeds his position size. Or this may be an issue when:
- user open a long position with 0.1x leverage, we are requiring IM ratio here
- user open a short position with 15x leverage, we only requiring MM ratio here, so this would succeed

Reply:
fixed in a36cff9